packages/react-input/src/components/Input/useInputStyles.ts

```diff
@@ -1,8 +1,18 @@
 import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import { tokens } from '@fluentui/react-theme';
-import type { InputState } from './Input.types';
+import type { InputSlots, InputState } from './Input.types';
+import { SlotClassNames } from '@fluentui/react-utilities/src';
 
+/**
+ * @deprecated
+ */
 export const inputClassName = 'fui-Input';
```

Comment:
For `Input` you can actually remove this class name instead of deprecating it since the component is still under `unstable`:

```suggestion
```